examples/relative_imports.rs

```diff
@@ -1,31 +1,20 @@
-use ruwren::{VMConfig, ModuleScriptLoader};
-
-struct TestLoader;
-
-impl ModuleScriptLoader for TestLoader {
-	fn load_script(&mut self, module: String) -> Option<String> {
-		if module == "test/a" {
-			Some(include_str!("relative_imports/a.wren").into())
-		} else if module == "test" {
-			Some(include_str!("relative_imports/test.wren").into())
-		} else {
-			None
-		}
-	}
-}
+use ruwren::{BasicFileLoader, VMConfig};
 
 fn main() {
-	let vm = VMConfig::new()
-		.enable_relative_import(true)
-		.script_loader(TestLoader)
-		.build();
+    let mut script_loader = BasicFileLoader::new();
```

Comment:
```suggestion
    let script_loader = BasicFileLoader::new().base_dir("examples/relative_imports");
```

Reply:
In this way the script_loader function doesn't accept it since it's not a closure anymore. I'm not sure how to solve this.

Reply:
https://github.com/Jengamon/ruwren/pull/8#discussion_r583161932 is how you'd resolve it. There isn't a point in making the builder `&mut self`, so I assumed it just took `self`. In the end, VMConfig takes ownership, so as is, it tries to take ownership of a `&mut Self`, which is no dice. Make the builder use `self` and this syntax should work.